admin/python/lsst/qserv/admin/logger.py

```diff
@@ -41,7 +78,7 @@ def setup_logging(path='logging.ini',
             logging.config.fileConfig(f)
         return True
     else:
-        logging.basicConfig(level=default_level)
+        logging.config.basicConfig(level=default_level)
```

Comment:
`logging.config` does not have documented `basicConfig` method, did you mean `dictConfig`?